view/theme/frio/templates/photo_view.tpl

```diff
@@ -51,12 +51,19 @@
 
 		{{* Tags and mentions *}}
 		{{if $tags}}
-		<div id="photo-tags">{{$tags.1}}</div>
+		<div id="photo-tags">{{$tags.title}}
+			{{foreach $tags.tags as $t}}
+			<span class="category label btn-success sm">
+				<span class="p-category">{{$t.name}}</span>
+				{{if $t.removeurl}} (<a href="{{$t.removeurl}}">x</a>) {{/if}}
```

Comment:
Please include the parentheses in the link and use a capital `X` for better readability.